src/useIsShake.ts

```diff
@@ -1,15 +1,19 @@
 import { useState, useEffect } from 'react';
 import { Accelerometer } from 'expo-sensors';
-import { Subscription } from 'expo-sensors/build/Pedometer';
+import { Subscription, isAvailableAsync, requestPermissionsAsync } from 'expo-sensors/build/Pedometer';
+import { openSettings } from 'expo-linking';
 
 type Poll = number | null;
 type IsShakeProps = { x: number };
+type AccelerometerStatus = 'uninitialized' | 'available' | 'not_available';
 
 export function useIsShake() {
   const [_, setData] = useState<IsShakeProps>();
   const [subscription, setSubscription] = useState<Subscription | null>(null);
   const [isShakeTriggered, setIsShakeTriggered] = useState(false);
   const [isShakeReady, setIsShakeReady] = useState(false);
+  const [accelerometerStatus, setAccelerometerStatus] = useState<AccelerometerStatus>('uninitialized');
```

Comment:
Using joined type instead of using `null`.

Reply:
WHAT A GREAT IDEA 🚀🚀🚀🚀🚀🚀🚀🚀